### ISSUE TITLE
Abnormal circulating Haptoglobin

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#4863]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
A decreased or increased levels of haptoglobin in the blood or serum

**Parent term (use hpo.jax.org/app)**
Abnormal circulating globulin level HP:0032179
http://purl.obolibrary.org/obo/HP_0032179

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Decreased haptoglobin levels occur with intravascular hemolysis or extravascular hemolysis. Increased levels in acute or chronic inflammatory response, tissue destruction, and malignant neoplasms
https://s.details.loinc.org/LOINC/4542-7.html?sections=Comprehensive
**Your nano-attribution (ORCID)**
0000-0001-9969-8610

**Comment by pnrobinson** [comment]
Is haptoglobin really a globulin ?

**Comment by nicolevasilevsky** [comment]
I don't think so

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/pubmed/24809098
